core/src/test/java/com/google/instrumentation/stats/StatsContextFactoryTest.java

```diff
@@ -24,8 +24,14 @@
  */
 @RunWith(JUnit4.class)
 public class StatsContextFactoryTest {
-  @Test(expected = IOException.class)
+  @Test
```

Comment:
So this no longer throws an exception? That's unexpected - is there a reason it shouldn't?